### ISSUE TITLE
StartSikulli process taking longer than timeout

### ISSUE THREAD

**Issue by andreguilhon** [rainmanwy/robotframework-SikuliLibrary#82]
Hello, I'm having some troubles when starting sikulli process in Windows 10 machines.
Sometimes, it happens that the start_sikuli_process KW "hangs", and I can only kill it by forcefully killing RF itself. 
I'm yet not sure, but it seems that the java process is still there. It does not respect timeout set when importing the library. 
My guess is that urlopen(url).read() in line 177 in sikulli.py is being hang, due to "blocking operations like the connection attempt".  If I'm correct, setting a timeout in this operation equal to the remaining time to timeout would avoid this issue.
I'm doing more investigation in this issue, so any tips, requests of logs, or something else are greatly appreciated.

**Comment by andreguilhon** [comment]
So far I found this out:
It is actually failing in the self.process.start_process call. It seems that RF is waiting forever for the process to start, but it actually started and is ready for use.

**Comment by andreguilhon** [comment]
That's correct. It doesn't happen all the time, just sometimes, which makes it harder to figure it out.
I've already check my browser to see if Sikuli is responding when the process hangs. It is. Every time. 
I'm using RF 3.0.2 and rf-sikuli 1.0.1.

**Comment by andreguilhon** [comment]
I'm using python3. And yes, I could access the web page. 
The logfiles are apparently clear. I'll get them next time it happens, but I can tell you that there's nothing abnormal in it. The error one is empty.
I'll get back here as soon as it happens again!

**Comment by rainmanwy** [comment]
thank you!  how about the jdk version?

i may check the implementation of start_process later!

获取 Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by andreguilhon** [comment]
PS C:\Windows\system32> java -version
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)

**Comment by andreguilhon** [comment]
@rainmanwy I only enabled log to see if I could get anything useful from it. It will hang with or without log.
